pyuvsim/simsetup.py

```diff
@@ -418,7 +418,7 @@ class SkyModelData:
     def __init__(self, sky_in=None):
         # Collect relevant attributes.
         if sky_in is not None:
-            self.name = sky_in.name
+            self.name = np.asarray(sky_in.name) if sky_in.name is not None else None
```

Comment:
I don't love one line if/else statements, I find them hard to read.

Reply:
oh, I love them when they're small like this, but that's OK. Easy fix!